qa/rpc-tests/sprout_sapling_migration.py

```diff
@@ -56,15 +56,15 @@ def __init__(self):
 
     def setup_nodes(self):
         extra_args = [[
+            '-allowdeprecated=z_getnewaddress',
+            '-allowdeprecated=z_getbalance',
         ]] * self.num_nodes
         # Add migration parameters to nodes[0]
         extra_args[0] = extra_args[0] + [
             '-migration',
             '-migrationdestaddress=' + SAPLING_ADDR,
             '-debug=zrpcunsafe'
         ]
-        assert_equal(3, len(extra_args[0]))
-        assert_equal(0, len(extra_args[1]))
```

Comment:
The reason for these assertions was to make sure that the inner arrays weren't aliased. Consider keeping them with the first argument changed to 5 and 2.